src/main/resources/swagger-ui/yaml/repo-org.yaml

```diff
@@ -114,6 +114,39 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+    head:
+      summary: Checks if repository settings exist
+      operationId: existUserRepo
+      tags:
+        - repository
+      parameters:
+        - name: uname
+          in: path
+          required: true
+          description: User name
+          schema:
+            type: string
+        - name: rname
+          in: path
+          required: true
+          description: Name of the repository
+          schema:
+            type: string
```

Comment:
same here

Reply:
Colleagues thank you for your useful wished to code.
@olenagerasimova, @dgarus Please review again